doc/news/slicing.rst

```diff
@@ -0,0 +1,3 @@
+**Added:**
+
+* Added additional methods to `collection.__getitem__()`, allowing for creating new collections from existing collections by providing a list of identifiers (`db["id1","id2"]`), integers (`db[0,2]`) or simply a slice (`db[2:3]`). Additionally, entries can now be selected by their position in the collection (`entry = db[3]`).
```

Comment:
Is the result of `db[0,2]` and `db[[0,2]]` the same?
Did I miss the test for `db[0,2]`?